core/templates/dev/head/header_css_libs.html

```diff
@@ -26,4 +7,6 @@
 <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet">
 <link rel="stylesheet" type="text/css" media="screen"
-      href="/third_party/static/font-awesome-4.4.0/css/font-awesome.min.css">
+      href="/css/oppia.css">
+<link rel="stylesheet" type="text/css" media="screen"
```

Comment:
manifest.json will also be used to download and unzip files in ```install_third_party.py``` so if developer want to test new css will have to specify it in ```manifest.json``` together with path to the  folder which contain ```css files``` . in gulpfile.js will create a list of css files to to be concatenated and minified. 
I Think I will specify just unminified files which will be concatenated and minified only if in prod. And yes it is possible to watch and reload if files change but I hesitated to watch third part library as I thought developer do not change codes in those files. But I think I will watch them.